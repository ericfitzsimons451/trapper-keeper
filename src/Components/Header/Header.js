import React from "react";
import { NavLink } from "react-router-dom";
import "./Header.scss";

export const Header = () => {
  return (
    <header className="header">
      <h1>Trapper Keeper</h1>
      <div className='new-note-style-container'>
        <NavLink to="/new-note" className='create-new'>Create New Note</NavLink>
      </div>
    </header>
  )
};