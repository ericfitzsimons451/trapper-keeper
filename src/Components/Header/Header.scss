header {
	display: flex;
	justify-content: space-between;
	flex-direction: row;
	border-bottom: 1px solid #e0e0e0;
	background: #fff;
  	// background-image: 
    // 	url("https://www.transparenttextures.com/patterns/dark-tire.png");
	h1 {
		color: #767676;
		text-align: left;
		margin-left: 1rem;
	}
}

.new-note-style-container {
	display: flex;
	align-items: center;
	justify-content: center;
}

.create-new {
	margin-right: 1rem;
	border: solid 1px;
	color: #767676;
	padding: 10px;
	border-radius: 15px;
	background: #fff;
	text-decoration: none;
}