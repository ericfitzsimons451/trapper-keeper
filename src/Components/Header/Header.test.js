import React from 'react';
import {shallow} from 'enzyme';
import { Header } from './Header'

describe('Header', () => {
	let wrapper;
	
	it('should match the snapshot', () => {
		wrapper = shallow(<Header />)
		expect(wrapper).toMatchSnapshot()
	})
})