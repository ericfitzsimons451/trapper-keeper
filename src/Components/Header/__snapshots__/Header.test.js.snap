// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div>
  Trapper Keeper
</div>
`;

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<div>
  I am a stateless header
</div>
`;
